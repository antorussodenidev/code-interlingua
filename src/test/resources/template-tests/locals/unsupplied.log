@ com.mikesamuel.cil.template.TemplateBundle$TemplateProcessingPass.postvisit
SEVERE: locals/C.java:15+34-35: Cannot interpolate null
