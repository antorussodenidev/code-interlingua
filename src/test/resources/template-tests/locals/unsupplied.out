package foo;
class C {
  int x = 1;
  public static final int F = (%Y);
  C() {
    this.x = 3;
  }
}